README.md

```diff
@@ -39,11 +39,13 @@ Getting started
 1. Change current directory to `snare` project directory
 2. `docker-compose build`
 3. `docker-compose up`
-
+4. Snare will start on 0.0.0.0 with port 80.
 More information about running `docker-compose` can be found [here.](https://docs.docker.com/compose/gettingstarted/)
 
 [Note : Cloner clones the whole website, to restrict to a desired depth of cloning add `--max-depth` parameter]
 
+In case of an error while running `docker-compose up`, check the availability port 80, if it is occupied then reffer to docker documentations to change the default port.
```

Comment:
availability **of**

And I suggest moving this note up, close to the docker section, because after note about cloning it seems weird

Reply:
Ok, got it :smiley:

Reply:
should i place it bellow step 6 of 'Steps to setup' because after that 'docker build instruction' begin